docs/artifacts/operation-patient-i-health-cards-issue.json

```
@@ -49,6 +49,12 @@
     "min": 0,
     "max": "*",
     "part": [{
+      "name": "vcIndex",
+      "use": "out",
+      "min": 0,
+      "max": "*",
```

There should be only one index per tuple

```suggestion
      "max": "1",
```

---

How come? If a resourceLink applies to more than one card, this is how you'd say that. Or are you suggesting to just repeat the resourceLink for each card? (I'm fine with that.)

---

Agree that's a good simplification 👍🏼

---

In the example that is part of this PR, it was listed as a single-valued element, not an array, so I thought this was what you were intending. I have no preference, but if this can be multi-valued then the example will need to be changed instead. I don't really have a preference of one way over the other.

---

Technical point -- in FHIR Parameters, even a 0..* element shows us as separate parameter entries, never an array of ints. i.e. it'd look like:

```
parameter: [{
  name: "x",
  valueInteger: 0
}, {
  name: "x",
  valueInteger: 1
}]
```

... so the example still did match the schema even for 0..*. But I've simplified per your suggestion, and agree with Pascal, this s fine!

---

Ahh, right!